src/cli/bunx_command.zig

```diff
@@ -498,16 +582,16 @@ pub const BunxCommand = struct {
             absolute_in_cache_dir,
         )) |destination| {
             const out = bun.asByteSlice(destination);
-            _ = try Run.runBinary(
+            try Run.runBinary(
                 ctx,
                 try this_bundler.fs.dirname_store.append(@TypeOf(out), out),
                 this_bundler.fs.top_level_dir,
                 this_bundler.env,
                 passthrough,
                 null,
             );
-            // we are done!
-            Global.exit(0);
+            // runBinary is noreturn
+            comptime unreachable;
```

Comment:
oh, it already is lol

Reply:
the comptime unreachable asserts runBinary is noreturn
and to make this easier to read, so no one can possibly think "oh what happens when this returns".

it could be argued not required. what do you think?

Reply:
let's keep it